simplenote.el

```diff
@@ -368,6 +368,8 @@ setting."
     ;; Don't switch mode when set via file cookie
     (when (eq major-mode (default-value 'major-mode))
       (funcall simplenote-notes-mode))
+    ;; Rename the buffer something useful
+    (rename-buffer (simplenote2--note-headline (buffer-string)))
```

Comment:
Hey @mmcclimon 

thanks for the PR.

But I think that there's some confusion:

you're using **simplenote2--note-headline** function, which is a function from this repo, **simplenote2.el**:

https://github.com/alpha22jp/simplenote2.el/blob/a0941475c5fd71a31280f3219424af4586288719/simplenote2.el#L545

whereas this repo is for simplenote.el, an older package.

Some history:

- @cefstat wrote simplenote.el
- I started to look after it
- Then @alpha22jp wrote simplenote2.el as it added more features

A while back, I've added this note to the packages' README:

https://github.com/dotemacs/simplenote.el/commit/e836fcdb5a6497a9ffd6bceddd19b4bc52189078

Have a look to see if you're really using simplenote2.el or simplenote.el. Then let me know if this PR is still relevant to this repo.

Thank you